Config/GlobalSettings.py

```diff
@@ -0,0 +1,5 @@
+DB_ADDRESS = "bolt://localhost:7687/neo4j"
```

Comment:
Are you sure you want to save the config in the code and not as an external file? 
It might make it easier to deploy on OS later using config maps

Reply:
you can just replace the folder later on with the same name and it works well in os, this is the way we already do config in my team